src/docfx/lib/github/GitHubUserCache.cs

```diff
@@ -138,13 +133,13 @@ public Task<(Error error, GitHubUser user)> GetByCommit(string authorEmail, stri
             if (string.IsNullOrEmpty(authorEmail))
```

Comment:
does that mean if the author email is empty, we can't get any user from github even using commit?

Reply:
yep

Reply:
we should be able to get user by login(got from commit) even if the author email is empty?